### PR TITLE
Add more Queues to sidekiq

### PR DESCRIPTION
**Pull request by alecslupu**
Fixes: https://github.com/decidim/decidim/issues/12279

### CODE REVIEW

File: config/sidekiq.yml [hunk 1]
```diff
@@ -8,3 +8,9 @@
 - metrics
 - exports
 - translations
+- close_meeting_reminder
+- conference_diplomas
+- newsletters_opt_in
+- reminders
+- user_report
+- vote_reminder
```
Comment (andreslucena) [inline config/sidekiq.yml:16]:
```suggestion
- vote_reminder
- block_user
```

File: config/sidekiq.yml
Comment (andreslucena) [inline config/sidekiq.yml:1]: The `decidim_events` queue is no longer used, so we could remove it

**Review by andreslucena — CHANGES_REQUESTED** [review]
Checking out https://github.com/decidim/decidim/tree/develop/decidim-generators/lib/decidim/generators/app_templates/sidekiq.yml.erb I found one missing and one that we could remove

**Review by andreslucena — APPROVED** [review]
👍🏽 LGTM 

Leaving @fblupi the final review and merge

**Review by fblupi — APPROVED** [review]
LGTM!

**Pull request merged**